### PR TITLE
Added scaling parameter

### PR DESCRIPTION
**Pull request by pescap**
Add and configure the scaling parameter.

### CODE REVIEW

File: deepxde/config.py [hunk 1]
```diff
@@ -198,3 +200,13 @@ def disable_xla_jit():
     This is equivalent with ``enable_xla_jit(False)``.
     """
     enable_xla_jit(False)
+
+
+def set_scaling(scaling_mode):
+    """Sets the scaling mode for data parallel acceleration.
+
+    Args:
+        scaling_mode (str): Whether 'weak' or 'strong'
```
Comment (lululxvi) [inline deepxde/config.py:209]: Explain what strong and weak strong is. Not all users know this.

File: deepxde/data/pde.py [hunk 2]
```diff
@@ -100,6 +100,10 @@ def __init__(
                 raise ValueError(
                     "Parallel training via Horovod only supports pseudo train distribution."
                 )
+            elif config.scaling == "strong":
```
Comment (lululxvi) [inline deepxde/data/pde.py:103]: `if config.scaling == "strong":`

File: deepxde/config.py [hunk 3]
```diff
@@ -12,12 +12,14 @@
 comm = None
 world_size = 1
 rank = 0
+scaling = None
```
Comment (lululxvi) [inline deepxde/config.py:15]:
```
# Data parallel
scaling = None
# Data parallel via Horovod
hvd = None
...
```

File: deepxde/config.py [hunk 4]
```diff
@@ -18,6 +20,7 @@
 
         hvd.init()
         world_size = hvd.size()
+        parallel_scaling = "weak"
```
Comment (lululxvi) [inline deepxde/config.py:23]: Move `parallel_scaling` inside the `if` below

File: deepxde/config.py [hunk 5]
```diff
@@ -198,3 +201,15 @@ def disable_xla_jit():
     This is equivalent with ``enable_xla_jit(False)``.
     """
     enable_xla_jit(False)
+
+
+def set_scaling(scaling_mode):
```
Comment (lululxvi) [inline deepxde/config.py:206]: `set_parallel_scaling`

Comment (pescap) [inline deepxde/config.py:206]:
```suggestion
def set_parallel_scaling(scaling_mode):
```

File: deepxde/config.py [hunk 6]
```diff
@@ -20,7 +22,7 @@
         world_size = hvd.size()
         if world_size > 1:
             from mpi4py import MPI
-
+            parallel_scaling = "weak"
```
Comment (lululxvi) [inline deepxde/config.py:25]: Format via black.

**Pull request merged**